src/Microsoft.ComponentDetection.Contracts/TypedComponent/VcpkgComponent.cs

```diff
@@ -0,0 +1,73 @@
+﻿using PackageUrl;
+
+namespace Microsoft.ComponentDetection.Contracts.TypedComponent
+{
+    public class VcpkgComponent : TypedComponent
```

Comment:
Can we add tests for this new detector? See other detector test samples in [Microsoft.ComponentDetection.Detectors.Tests](https://github.com/microsoft/component-detection/tree/main/test/Microsoft.ComponentDetection.Detectors.Tests).